Sources/NIOPosix/SelectableEventLoop.swift

```diff
@@ -526,6 +557,7 @@ Further information:
                 }
 
                 // Execute all the tasks that were submitted
+                tasksProcessedInTick += self.tasksCopy.count
```

Comment:
We should be a little careful here. It's not really practically possible, but conceptually this addition can trap. We probably want to replace this with an `addingReportingOverflow` that, if we overflow, saturates instead.